production/mxnet_18.10-py3/Dockerfile

```diff
@@ -26,6 +41,9 @@ RUN apt-get install -y libsqlite3-dev \
 RUN python3 -m pip --version
 RUN python3 -m pip install -U setuptools
 
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python2.7 1
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.5 2
```

Comment:
Move RUN update-alternatives fixed system wide python version, and prevent auth-client-config runtime error when docker building image.